doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -119,7 +119,14 @@ This is a reference configuration that can be modified in the production firmwar
      - CRACEN + Oberon [2]_
      - Yes
      - Trusted Storage backend
-     - SHA-256 hash
+     - Hardware Unique Key (HUK)
+   * - nRF54L15 SoC + Trusted Firmware-M (TF-M)
+     - Thread
+     - PSA
+     - CRACEN
+     - Yes
+     - Trusted Firmware-M (TF-M)
```

Comment:
What do you think to adjust the table a bit:

1) Instead of Trusted Storage backend - just have "PSA Secure Storage backend" column. Where we can have either Trusted Storage library, TF-M secure storage service or none (-).
2) Cryptographic API: PSA -> PSA Crypto API
3) I'm not sure what to do with column "Trusted Storage AEAD key" - as for TF-M builds, it's quite a bit different and then we have options to store the key in ITS or KMU.